srcpkgs/python3-gitea-client/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'python3-gitea-client'
+pkgname=python3-gitea-client
+version=1.0.9
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3-future python3-requests python3-attrs"
+checkdepends="$depends python3-pytest python3-responses"
+short_desc="Python3 library for interacting with a gitea server"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://github.com/unfoldingWord-dev/python-gitea-client"
+distfiles="${PYPI_SITE}/g/gitea_client/gitea_client-${version}.tar.gz"
+checksum=6114e81bcec638f089dd84e374fe0c57981ec550386f7ba5aa1343409cfe7a18
+
+post_install() {
+	vlicense LICENSE.txt
+	# remove tests directory polluting site-packages
+	rm -rf ${DESTDIR}/usr/lib/python*/site-packages/tests
```

Comment:
```suggestion
	rm -r ${DESTDIR}/usr/lib/python*/site-packages/tests
```
If they change this upstream, we would want to know this rm doesn't remove anything anymore and install can only run once.